### DISCUSSION TITLE
Bug - strange rendering

### DISCUSSION THREAD

**Discussion by APerricone** [SixLabors/Fonts#375] [Q&A] [answered]
Hello, I am using this library to render an Italian text with the font [Permanent Marker](https://fonts.google.com/specimen/Permanent+Marker) the text is "È mezzogiorno" and I see that on "È" it goes down 
![image](https://github.com/SixLabors/Fonts/assets/22457168/2dc78e89-b39c-46c6-a150-6956fae09c1d)
If I try to use a C instead the text is rendered on the correct spot:
![image](https://github.com/SixLabors/Fonts/assets/22457168/f1bd6b7b-b5dc-4388-a570-72c2f8fd23a3)
I tried to call TextMeasurer methods and I see that at this letter it has very strange results.
can I apply a realtime correction?

**Comment by JimBobSquarePants** [comment]
Ah... You mean the whole line is pushed down. It took me a few reads to understand. That's definitely a bug. Can you please raise an issue providing as much information as possible to replicate. Thanks!

**Reply by APerricone** [reply]
You can download the woff2 file from google font, I use it as documented:
```c#
FontCollection fontCollection = new FontCollection();
FontFamily fontFamily=fontCollection.Add("PermanentMarker.woff2");
var font = fontFamily.CreateFont(142);
var txtOpt =new TextOptions( font );
var m1 = TextMeasurer.MeasureAdvance( "È", txtOpt );
var m2 = TextMeasurer.MeasureBounds( "È", txtOpt );
var m3 = TextMeasurer.MeasureSize( "È", txtOpt );
var m4 = TextMeasurer.MeasureAdvance( "C", txtOpt );
var m5 = TextMeasurer.MeasureBounds( "C", txtOpt );
var m6 = TextMeasurer.MeasureSize( "C", txtOpt );
```
Here I compared the measurement value with the uppercase C because they look similar:
![image](https://github.com/SixLabors/Fonts/assets/22457168/12610501-6756-42f6-888b-d9681bbc0c39)
(sorry, it uses comma for decimal values)
The values are pretty different and for uppercase accented e are pretty strange,  like heights near 4000!

I am using .NET 6.0 , SixLabors.Fonts 2.0.1 WITHOUT ImageSharp. Visual Studio 2022

**Reply by JimBobSquarePants** [reply]
What are you using to render the text in your screenshots if not ImageSharp.Drawing?

**Reply by APerricone** [reply]
I am using [PDFSharp](https://docs.pdfsharp.net/) to create PDF file with a derived IGlyphRenderer because I need the text are paths instead of text. I am writing a character a time so this problem is more visible. The screenshot are from Acrobat reader

**Reply by JimBobSquarePants** [reply]
I have a fixed the woff2 reader #380 so that bounds now match the TTF version. I also added fixes for a few other things I discovered during testing, notably that the size was not reflective of the bounds.

**Reply by JimBobSquarePants** [reply, marked as answer]
Fix is released.
